src/Microsoft.ApplicationInsights.AspNetCore/RequestTrackingMiddleware.cs

```diff
@@ -1,64 +0,0 @@
-﻿namespace Microsoft.ApplicationInsights.AspNetCore
-{
-    using System;
-    using System.Diagnostics;
-    using System.Threading.Tasks;
-
-    using Microsoft.ApplicationInsights;
-    using Microsoft.ApplicationInsights.AspNetCore.Extensions;
-    using Microsoft.ApplicationInsights.DataContracts;
-    using Microsoft.ApplicationInsights.Extensibility.Implementation;
-    using Microsoft.AspNetCore.Builder;
-    using Microsoft.AspNetCore.Http;
-
-    /// <summary>
-    /// Sends telemetry about requests handled by the application to the Microsoft Application Insights service.
-    /// </summary>
-    public sealed class RequestTrackingMiddleware
```

Comment:
Do you want to keep a class or just an empty extension method on `IApplicationBuilder`, because that's what most of the customers used.